code/modules/library/book.dm

```diff
@@ -123,25 +123,24 @@
 
 /obj/item/book/attackby(obj/item/I, mob/user, params)
 	if(istype(I, /obj/item/pen))
+		if(!user.canUseTopic(src, BE_CLOSE) || !user.can_write(I))
+			return
 		if(user.is_blind())
 			to_chat(user, span_warning("As you are trying to write on the book, you suddenly feel very stupid!"))
 			return
```

Comment:
do you need this part anymore with the can_write check above?

Reply:
`can_write()` does not check for blindness.  There is a reason blind people use braille to write with since the dots are indention marks that they feel with.